src/cgame/rocket/rocketFormControlSelect.h

```diff
@@ -119,22 +117,24 @@ class CvarElementFormControlSelect : public Rml::ElementFormControlSelect, publi
 	void UpdateValue()
 	{
 		Rml::String cvarValue = Cvar::GetValue( cvar.c_str() );
+
 		for ( int i = 0; i < GetNumOptions(); ++i )
 		{
 			Rml::Element* e = GetOption(i);
 			Rml::String value = e->GetAttribute<Rml::String>( "value", Rml::String() );
 			if ( value == cvarValue )
 			{
-				// If the selection is already correct, do not force set the cvar.
-				ignore_value_change = GetSelection() != i;
+				ignore_value_change = true;
 				SetSelection( i );
+				ignore_value_change = false;
```

Comment:
why we need to do this? Does setselection trigger an event? maybe comment this?

Reply:
Added comment on ignore_value_change.